spark-commons/src/main/scala-spark2-jvm/za/co/absa/spark/commons/adapters/TransformAdapter.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spark.commons.adapters
+
+import org.apache.spark.sql.Column
+import za.co.absa.spark.hofs.{transform => hofsTransform}
+
+trait TransformAdapter {
+  /**
+   * For Spark versions prior 3.0.0, delegates to {{{hofs.transform()}}}
+   */
+  def transform: (Column, Column => Column) => Column = hofsTransform
```

Comment:
Wondering if to include `SparkVersionGuard` call here. 🤔

Reply:
Don't think it's needed. I also think the comments are not so necessary

Reply:
Don't think it's needed. I also think the comments are not so necessary

Reply:
Agreed not needed. At least not for now.